functions/src/events/scrapeEvents.ts

```diff
@@ -138,12 +144,55 @@ class HearingScraper extends EventScraper<HearingListItem, Hearing> {
     return events.filter(HearingListItem.guard)
   }
 
-  async getEvent({ EventId }: HearingListItem) {
-    const content = HearingContent.check(await api.getHearing(EventId))
+  async getEvent({ EventId }: HearingListItem /* e.g. 4962 */) {
+    const data = await api.getHearing(EventId)
+    const hearing = Hearing.check(data)
+    const content = HearingContent.check(data)
+
+    let maybeVideoURL = null
+    let transcript = null
+    if (!hearing.videoFetchedAt) {
+      const req = await fetch(
+        `https://malegislature.gov/Events/Hearings/Detail/${EventId}`
+      )
+      const res = await req.text()
+      if (res) {
+        const dom = new JSDOM(res)
+        if (dom) {
+          const maybeVideoSource =
+            dom.window.document.querySelectorAll("video source")
+          if (maybeVideoSource.length && maybeVideoSource[0]) {
+            const firstVideoSource = maybeVideoSource[0] as HTMLSourceElement
+            maybeVideoURL = firstVideoSource.src
+
+            transcript = await assembly.transcripts.transcribe({
```

Comment:
If we're using the webhook approach, should this use `transcripts.submit` instead of `transcripts.transcribe`?

Reply:
good call, updating in next commit.